noxfile.py

```diff
@@ -379,6 +383,98 @@ def check_bundled_app(session: nox.Session) -> None:
             )
 
 
+@nox.session()
+def html2jpeg(session: nox.Session):
+    """Create JPEGs from HTML for documentation"""
+
+    port = 12305
+
+    def exporter_running():
+        with closing(socket.socket(socket.AF_INET, socket.SOCK_STREAM)) as sock:
+            if sock.connect_ex(("localhost", port)) == 0:
+                return True
+            return False
+
+    if not exporter_running():
+        session.error(
+            f"HTML exporter not running. Please run 'docker run  -p {port}:2305 bedrockio/export-html'"
+        )
+
+    for html, jpeg, size in [
+        (
+            "doc/examples/example_html.html",
+            "doc/images/screenshot-html.jpeg",
+            [800, 290],
+        ),
+        (
+            "doc/examples/example_html.details.example.cpp.9597a7a3397b8e3a48116e2a3afb4154.html",
+            "doc/images/screenshot-html-details.example.cpp.jpeg",
+            [800, 600],
+        ),
+        (
+            "gcovr/tests/html-themes/reference/gcc-5/coverage.green.main.cpp.118fcbaaba162ba17933c7893247df3a.html",
+            "doc/images/screenshot-html-default-green-src.jpeg",
+            [800, 290],
+        ),
+        (
+            "gcovr/tests/html-themes/reference/gcc-5/coverage.blue.main.cpp.118fcbaaba162ba17933c7893247df3a.html",
+            "doc/images/screenshot-html-default-blue-src.jpeg",
+            [800, 290],
+        ),
+        (
+            "gcovr/tests/html-themes-github/reference/gcc-5/coverage.green.main.cpp.118fcbaaba162ba17933c7893247df3a.html",
+            "doc/images/screenshot-html-github-green-src.jpeg",
+            [800, 500],
+        ),
+        (
+            "gcovr/tests/html-themes-github/reference/gcc-5/coverage.blue.main.cpp.118fcbaaba162ba17933c7893247df3a.html",
+            "doc/images/screenshot-html-github-blue-src.jpeg",
+            [800, 500],
+        ),
+        (
+            "gcovr/tests/html-themes-github/reference/gcc-5/coverage.dark-green.main.cpp.118fcbaaba162ba17933c7893247df3a.html",
+            "doc/images/screenshot-html-github-dark-green-src.jpeg",
+            [800, 500],
+        ),
+        (
+            "gcovr/tests/html-themes-github/reference/gcc-5/coverage.dark-blue.main.cpp.118fcbaaba162ba17933c7893247df3a.html",
+            "doc/images/screenshot-html-github-dark-blue-src.jpeg",
+            [800, 500],
+        ),
+    ]:
+
+        def read_file(file):
+            with open(file, encoding="utf-8") as fh_in:
+                return " ".join(fh_in.readlines()).replace("\n", "")
+
+        content = re.sub(
+            r'<link rel="stylesheet" href="([^"]+)"/>',
+            lambda match: f'<style type="text/css">{read_file(os.path.join(os.path.dirname(html), match[1]))}</style>',
+            read_file(html),
+        )
+        data = {
+            "html": content,
+            "export": {
+                "type": "jpeg",
```

Comment:
The documentation claims that `"type": "png"` should work, which would be preferable for screenshots: https://github.com/bedrockio/export-html

Reply:
There is an [issue](https://github.com/bedrockio/export-html/issues/1) that the quality is always set but not supported for PNG.